RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHETimeSelected_cfi.py

```diff
@@ -1,4 +1,4 @@
-import FWCore.ParameterSet.Config as cms
+1;2802;0cimport FWCore.ParameterSet.Config as cms
```

Comment:
Something weird happened here.  Could you remove the extra characters and update?